storage/src/main/java/org/apache/uniffle/storage/handler/impl/HdfsClientReadHandler.java

```diff
@@ -143,7 +143,7 @@ protected void init(String fullShufflePath) {
           LOG.warn("Can't create ShuffleReaderHandler for " + filePrefix, e);
         }
       }
-      readHandlers.sort(Comparator.comparing(HdfsShuffleReadHandler::getFilePrefix));
+      Collections.shuffle(readHandlers);
```

Comment:
Could we print the file names? It's important for us to know the order.

Reply:
Done